doc.go

```diff
@@ -18,5 +18,92 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// Package zap provides fast, structured, leveled logging in Go.
+// Package zap provides fast, structured, leveled logging.
```

Comment:
Errrr...wow, I hadn't thought of that. I don't know of one.

If this comes up in an issue or something, I'll dig up a link and add an FAQ entry.